### ISSUE TITLE
Wording recommendations

### ISSUE THREAD

**Issue by njx** [adobe/brackets-edge-web-fonts#10]
Might need to tweak these further after seeing how they sit in the various dialogs. Might be too wordy.
## Code hint menu

Can we say "Browse Edge Web Fonts..." instead of "Browse EWF..."? If that's too long, maybe we could just have it say "Edge Web Fonts..."
## Informational Wf dialog

This should show when you're in an HTML file or a CSS file that has no EWFs referenced yet (should check with marketing on the messaging in the first sentence; also, would be cool if we could stick the actual Wf icon where it says "the 'Wf' icon" below)

_Dialog title:_

> How to Use Edge Web Fonts

_Dialog body:_

> Edge Web Fonts are a collection of free online fonts that you can easily use on your website.
> 
> To add an Edge Web Font to your CSS file, type **Ctrl-Space** while in a `font-family:` attribute, 
> then click on "Browse Edge Web Fonts..."
> 
> After adding fonts to your CSS file, **click on the "Wf" icon** in the upper right-hand corner of the
> {APP_NAME} window. This will give you a `<script>` tag that you can paste into your HTML file 
> to load the Edge Web Fonts you're using.
## Wf dialog that shows the script tag

_Dialog title:_

> Generate Edge Web Fonts Include Code

_Dialog body:_

> Edge Web Fonts are loaded into the user's browser via JavaScript. To ensure that fonts are 
> properly loaded, you must add a script tag to your HTML.
> 
> Copy the following script tag and paste it into all HTML files that reference this CSS file:
## Text inside EWF browse dialog

(again, would be good if we could show the Wf icon here; also, maybe Garth can make a nice "warning" or "important info" icon we could put to the left of it)

> **Important:** After adding fonts to your CSS file, **click on the "Wf" icon** in the upper right-hand 
> corner of the {APP_NAME} window. This will give you a `<script>` tag that you can paste into 
> your HTML file to load the Edge Web Fonts you're using.

**Comment by njx** [comment]
Would be nice to reword "Generate Edge Web Fonts Include Code" to something that's a little easier to parse, but I can't think of anything obvious.

**Comment by adrocknaphobia** [comment]
Doh! I didn't see this issue when I started working on the dialogs. Pull down the latest and let me know what you think of my version of "How To".

**Comment by njx** [comment]
The image is a great idea--we should definitely keep it. The one thing that might be missing from it is that it doesn't really represent the step where the font-family gets added to the CSS file (it shows the browse dialog, but not that that results in CSS being written). But I think that's okay...the important thing is to show the overall workflow. You might want to have @GarthDB take a look at it in case he wants to make any tweaks to the visual design.

I'd suggest changing the dialog title and the steps at the bottom to be closer to what I suggested, though. Some specific reasons:
- It's worth bolding the specific steps the user has to do so their eye is drawn to them if they're skimming. (I also thought it would be good to add the Ctrl-Space note, since there are cases where the popup doesn't come up automatically, like if you're adding a font to the front of an existing font-family value--which might happen if they're adding fonts to an existing CSS file.)
- We should include the actual Wf icon in the text, again for quick recognition if they're just skimming.
- I suggested we specifically say "After adding fonts to your CSS file" instead of something more generic (like "When you're done") so it's clear that you should click the Wf icon after you've added whatever fonts you want to add.
- I thought it would be better to not explicitly describe the pasting step as a separate step, since there will be guidance in the Wf dialog about that anyway, and I didn't want the user to feel like s/he had to memorize three things. That said, I have an extra sentence about it anyway (it's just glued onto the second step), so maybe it's not that different.

If we do keep something closer to your current wording, some notes:
- The property is font-family, not font-face (you don't have to write the @font-face rules yourself--the script does that dynamically).
- Should use double-quotes (not single-quotes) around "Browse Web Fonts". Also, the name of the menu item is currently "Browse EWF...", though I'm proposing we change it to "Browse Edge Web Fonts..." However we change it, the instructions should be consistent.
- I don't think we need to say "free online web fonts" again in the first step since we emphasize it in the intro paragraph.
- In step 2, "your" should be "you're".
- We should say "in the top right of the {APP_NAME} window" so the user doesn't think we mean the upper right of the Browse Edge Web Fonts dialog.
- We shouldn't say "embed" in step 2 since that might sound like it's going to create an <embed> tag. I think we can just say "`<script>` tag".

**Comment by njx** [comment]
Actually, I take back what I said about two steps vs three. In your version, the three steps are nicely tied to the image...I didn't get that right away. So let's keep your structure but maybe tweak it as above. Also, consider adding little (1)/(2)/(3) bubbles to the image to make it obvious that they match the steps. 

Also, in the last image, it might be good to actually show a snippet of HTML code at normal font size--right now it's actually not visually obvious that it's an HTML file.

**Comment by njx** [comment]
Okay, maybe I should shut up until I've thought things through :) I just realized that the second step doesn't really line up with the image, because the image shows the dialog, while the second step talks about clicking the Wf icon (which actually really is important for people to remember, and isn't represented in the image). Not sure how we would wedge it in without making the image too complicated though.

Anyway, I'm sure I'm overthinking this. Feel free to take any of the above feedback that you think is appropriate :)

**Comment by pthiess** [comment]
Assigning to Adam to decide what needs to be done for Sprint 14.

**Comment by pthiess** [comment]
@adrocknaphobia Adam, please review these proposals and file separate bugs for anything we need to address within this week.

**Comment by joelrbrandt** [comment]
@adrocknaphobia can we close this bug? Marking as FBNC for now, since our strings are final for the release.

**Comment by iwehrman** [comment]
@ryanstewart Any thoughts on this? Is this fixed?

**Comment by ryanstewart** [comment]
Yup, marking as fixed.
